components/FlexibleHeader/src/private/MDCStatusBarShifter.m

```diff
@@ -205,12 +223,12 @@ - (void)setOffset:(CGFloat)offset {
     return;
   }
 
-  // Bound the status bar range to [0...kStatusBarExpectedHeight].
-  CGFloat statusOffsetY = MIN(kStatusBarExpectedHeight, offset);
+  // Bound the status bar range to [0...statusBarHeight].
```

Comment:
Question: Should this comment match the variable (`_originalStatusBarHeight`) or is it intentionally different?

Reply:
Fixed